Improve docs about dynamic endpoint parameters (#1279)
wso2/docs-choreo-dev: en/docs/develop-components/configure-endpoints-body.md

### Configure endpoints with the Ballerina buildpack

> When you create a service component with the `Ballerina buildpack`, Choreo automatically detects the endpoint details for REST APIs. You can override the auto-generated endpoint configuration by providing the `endpoints.yaml` file in the source directory. For dynamic endpoint parameters, such as variable ports, automatic endpoint generation is not supported. To address this, use an endpoint.yaml file as a configuration solution to define dynamic parameters.

Reviewer: ```suggestion
When you create a service component with the `Ballerina buildpack`, Choreo automatically detects the endpoint details for REST APIs. You can override the auto-generated endpoint configuration by providing the `endpoints.yaml` file in the source directory.

!!! note
    Automatic endpoint generation is not supported for dynamic endpoint parameters such as variable ports. Therefore, you must use an `endpoint.yaml` file to define dynamic endpoint parameters.
```

Author: Fixed in https://github.com/wso2/docs-choreo-dev/pull/1279/commits/619cbf15c8fcce39406112376a1fec821fcd2405
